openscm/highlevel.py

```diff
@@ -20,7 +20,9 @@ class OpenSCM(Core):
 class ScmDataFrame(ScmDataFrameBase):
     """OpenSCM's custom DataFrame implementation.
 
-    The ScmDataFrame wraps around `pyam`'s IamDataFrame, which itself wraps around Pandas.
+    The ScmDataFrame implements a subset of the functionality provided by `pyam`'s
+    IamDataFrame, but is focused on the providing a performant way of storing
+    time series data and the metadata associated with those time series.
```

Comment:
For users who wish to take advantage of all of Pyam's functionality, please cast your data frame to an IamDataFrame first with `to_iamdataframe()`. Note: this operation can be relatively computationally expensive for large data sets.